packages/design-system/src/components/web-components/ds-month-picker/ds-month-picker.stories.tsx

```diff
@@ -0,0 +1,173 @@
+      control: 'text',
+    },
+    'clear-all-text': {
+      description: 'The text for the "Clear all" button for internationalization.',
+      control: 'text',
+    },
+    'default-selected-months': {
+      description:
+        'Array of month numbers, where `1` is January, and any month included is selected by default. Sets the initial checked state for the 12 month checkboxes. Use this for an uncontrolled component; otherwise, use the `selectedMonths` property. Alternatively, you can apply `checked` to `<input>` element children.',
+      control: 'text',
+    },
+    'disabled-months': {
+      description:
+        'Array of month numbers, where `1` is January, and any month included is disabled for selection. Alternatively, you can apply `disabled` to `<input>` element children.',
+      control: 'text',
+    },
+    'error-id': {
+      description:
+        'The ID of the error message applied to this field. If none is provided, the id will be derived from the `root-id` attribute.',
+      control: 'text',
+    },
+    'error-message-class-name': {
+      description: 'Additional classes to be added to the error message.',
+      control: 'text',
+    },
+    'error-message': {
+      description: 'Enable the error state by providing an error message.',
+      control: 'text',
+    },
+    'error-placement': {
+      description: 'Location of the error message relative to the field input',
+      options: [undefined, 'top', 'bottom'],
+      control: { type: 'radio' },
+    },
+    'hint-id': {
+      description: 'The ID of the hint element',
+      control: 'text',
+    },
+    hint: {
+      description: 'Hint text or HTML',
+      control: 'text',
+    },
+    inversed: {
+      description: 'Set to "true" to apply the "inverse" scheme',
+      control: 'boolean',
+    },
+    'label-class-name': {
+      description: 'Additional classes to be added to the field label',
+      control: 'text',
+    },
+    'label-id': {
+      description:
+        "A unique `id` to be used on the field label. If one isn't provided, a unique ID will be generated.",
+      control: 'text',
+    },
+    label: {
+      description: 'Label text or HTML.',
+      control: 'text',
+    },
+    name: {
+      description: "The `input` field's `name` attribute.",
+      control: 'text',
+    },
+    'requirement-label': { control: 'text' },
+    'root-id': {
+      description:
+        "A unique ID for this element. A unique ID will be generated if one isn't provided.",
+      control: 'text',
+    },
+    'select-all-text': {
+      description: 'The text for the "Select all" button for internationalization.',
+      control: 'text',
+    },
+    'selected-months': {
+      description:
+        'Array of month numbers, where `1` is January, and any month included is selected. This will render a read-only field. If the field should be mutable, use `defaultSelectedMonths`. Alternatively, you can apply `checked` to `<input>` element children.',
```

Comment:
Since it would be weird behavior anyway to not allow a user to change something that isn't disabled, I think we should leave it, and maybe we should actually remove the `selected-months` attribute from the web component entirely.

Reply:
removing `selected-months`!